x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/synthetics/kpi_over_time_config.ts

```diff
@@ -109,6 +109,18 @@ export function getSyntheticsKPIConfig({ dataView }: ConfigProps): SeriesConfig
           },
         ],
       },
+      {
+        label: 'Monitor Complete',
+        id: 'state.up',
+        field: 'state.up',
+        columnType: OPERATION_COLUMN,
+        columnFilters: [
+          {
+            language: 'kuery',
+            query: `state.id: * and state.up: 1`,
```

Comment:
```suggestion
            query: `summary: * and summary.down: 0 and monitor.status: "up" `,
```

I think state part is still in progress so i am bit hesitant to use that here. So let's use summary. 

summary.down: 0 means there is no down check. but it also means monitor might error out in case of browser monitors. so we need an additional filter, either use monitor.status: "up" or `summary.up > 1 `

It's kinda weird that we are doing these filters. it's because of the way summary document is generated. summary.up: 1 doesn't means it's up, there can be cases where `summary.up: 1` and it can also have `summary.down: 1` 

So complete check is where there is now down check and also status should be up. 

I feel like this is worth discussing in next tech sync :)

Reply:
I copy/pasted the definition from the previous metric. Should that one be corrected as well?

Tangential to this: do we have the ES mappings documented anywhere with what each field means?